manifests/install.pp

```diff
@@ -15,13 +15,16 @@
 define kmod::install(
   $ensure=present,
   $command='/bin/true',
-  $file='/etc/modprobe.d/modprobe.conf',
+  $file="/etc/modprobe.conf/${title}.conf",
```

Comment:
I'd use `$name` here too, as it's more likely to be the relevant file name if the end user wants to take advantage of the namevar.